docs/notebook_validation.py

```diff
@@ -0,0 +1,75 @@
+# ============================================================================ #
+# Copyright (c) 2022 - 2023 NVIDIA Corporation & Affiliates.                   #
+# All rights reserved.                                                         #
+#                                                                              #
+# This source code and the accompanying materials are made available under     #
+# the terms of the Apache License 2.0 which accompanies this distribution.     #
+# ============================================================================ #
+
+import glob, os, re, sys
+from pathlib import Path
+import subprocess
+from shutil import which
+if which('jupyter') is None:
+    print("Please install jupyter, e.g. with `pip install notebook`.")
+    exit(1)
+
+def available():
+    available_backends = sys.stdin.readlines()
+    if available_backends:
+        available_backends = [
+            available_backend.strip()
+            for available_backend in available_backends
+        ]
+        return available_backends
+
+def validate(notebook_filename):
```

Comment:
I think you meant update the validate function here to call execute.

Reply:
yeah, i forgot to call execute function